provision-set-edx-theme.sh

```diff
@@ -2,7 +2,7 @@
 
 # This script sets up the edX theme in LMS and CMS.
 
-REPO_URL="https://github.com/edx/edx-themes"
+REPO_URL="git@github.com:edx/edx-themes.git"
```

Comment:
By default `make dev.clone` moved to cloning via SSH. This updates the provisioning script to do the same.